File: Workloads/len.py
from time import time


def workload(list):
    list.__len__()


len = 14000
totalTime = 0
runs = 8
for run in range(runs):
    list = [i for i in range(len)]
    start = time()
    workload(list)
    totalTime += time() - start


print(totalTime/runs)

File: java_tests/org/python/java_tests/workloads/Len.java
package org.python.java_tests.workloads;

public class Len {
	public static void main(String[] args) {
		int len = 14000;
		int totalTime = 0;
		int runs = 8;
		org.python.types.List list = new org.python.types.List();
		
		for(int i= 0; i<len; i++) {
			list.append(org.python.types.Int.getInt(i));	
		}
		
		for(int i = 0; i<runs; i++) {
			long startTime = System.currentTimeMillis();
			list.__len__();
			totalTime += System.currentTimeMillis() - startTime;
		}

    	System.out.println("Time in miliseconds " + (totalTime/runs));
	}
}